src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form search-form_big"
```

Comment:
```suggestion
      class="search-form"
```
pay attention on this point in [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
same for other `big` elements
![Screenshot from 2023-09-09 17-57-50](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/05c4af15-9f83-4b17-b5b1-c33604d01956)